src/components/navigation/navbar/index.tsx

```diff
@@ -88,20 +71,15 @@ const Navbar = () => {
             backgroundColor: 'white',
           }}
         >
-          <StatusIndicatorDot color="yellow" />
-          {userName && userName}
+          {isDisconnected ? (
+            <Text>Not connected</Text>
```

Comment:
this is fine but if they are not connected they shouldn't be able to navigate to badge directory screen.